src/private/WRLDRoute.mm

```diff
@@ -0,0 +1,12 @@
+#import "WRLDRoute.h"
+
+@interface WRLDRoute ()
```

Comment:
These data bag types all have pretty much empty .mms, any way we can remove these? If not fine to leave for now